Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "socket-server-mocker"
 description = "Mock socket server in Rust, for testing various network clients."
 authors = ["Thomas Prévost"]
-version = "0.2.1"
+version = "0.3.0"
```

Comment:
Set the same version in README.md please

Reply:
good catch, thx, fixed